.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,42 @@
+# A CI configuration to check for licenses in Dart files.
+
+name: Changelog Check
+
+# Callers of this workflow should use it as follows:
+#
+# name: License Check
+# on:
+#   pull_request:
+#     branches: [ main ]
+#     types: [opened, synchronize, reopened, labeled, unlabeled]
+#   push:
+#     tags: [ 'v[0-9]+.[0-9]+.[0-9]+' ]
+# jobs:
+#   check_licenses:
+#     uses: dart-lang/ecosystem/.github/workflows/check_licenses.yml@main
+
+jobs:
+  changelog:
+    # These permissions are required for authentication using OIDC and to enable
+    # us to create comments on PRs.
+    permissions:
+      id-token: write
```

Comment:
We don't need the id-token one here - that's just something we need when trying to auto-publish a package.